app/src/main/java/org/p2p/wallet/jupiter/repository/tokens/JupiterSwapTokensRemoteRepository.kt

```diff
@@ -3,29 +3,27 @@ package org.p2p.wallet.jupiter.repository.tokens
 import timber.log.Timber
 import java.util.concurrent.TimeUnit
 import kotlinx.coroutines.withContext
-import org.p2p.core.utils.Constants.USD_READABLE_SYMBOL
+import org.p2p.core.crypto.Base58String
 import org.p2p.wallet.common.date.toDateTimeString
 import org.p2p.wallet.common.date.toZonedDateTime
-import org.p2p.wallet.home.model.TokenPrice
 import org.p2p.core.dispatchers.CoroutineDispatchers
 import org.p2p.wallet.infrastructure.swap.JupiterSwapStorageContract
 import org.p2p.wallet.jupiter.api.SwapJupiterApi
 import org.p2p.wallet.jupiter.api.response.tokens.JupiterTokenResponse
 import org.p2p.wallet.jupiter.repository.model.JupiterSwapToken
 import org.p2p.wallet.user.repository.UserLocalRepository
-import org.p2p.wallet.user.repository.prices.TokenCoinGeckoId
-import org.p2p.wallet.user.repository.prices.TokenPricesRemoteRepository
-import org.p2p.core.crypto.Base58String
 import org.p2p.core.crypto.toBase58Instance
+import org.p2p.token.service.interactor.TokenServiceInteractor
+import org.p2p.token.service.model.TokenServiceNetwork
+import org.p2p.token.service.model.TokenServicePrice
 
 internal class JupiterSwapTokensRemoteRepository(
     private val api: SwapJupiterApi,
     private val localRepository: JupiterSwapTokensLocalRepository,
     private val dispatchers: CoroutineDispatchers,
     private val userRepository: UserLocalRepository,
-    private val pricesRepository: TokenPricesRemoteRepository,
     private val swapStorage: JupiterSwapStorageContract,
-    private val pricesCache: JupiterSwapTokensPricesLocalRepository
+    private val tokenServiceInteractor: TokenServiceInteractor
```

Comment:
And again i'll raise this question
do we need interactor in such cases? it's just a proxy for Repository, we don't really have any domain logic for just fetching prices.

Reply:
and sure as strange to inject interactor inside the repo

Reply:
fix it